src/registries/is-request-to-registry.js

```diff
@@ -2,18 +2,25 @@
 
 import url from 'url';
 
+const SUFFIX_VISUALSTUDIO = '.pkgs.visualstudio.com';
```

Comment:
I think I missed that detail earlier.
It looks hard to scale a maintain a solution where every registry has to be hardcoded in Yarn.
Can this setting be moved to .yarnrc for each registry to configure?